### PR TITLE
Initial batch of sender signals

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

**Comment by ianthiel** [comment]
@morriscode yo! if you add the Notion Task link in the PR description, the PRs will get connected to that Task in Notion! I've connected these signal PRs to [this Task](https://www.notion.so/sublimesecurity/Merge-first-50-non-external-function-Signals-for-rule-testing-and-risk-score-to-main-f423b6a5cb064e8a892e364eef78d1f8?pvs=4) manually for now 👍

**Comment by morriscode** [comment]
> 

Oh Amazing!!! thank you

File: signals/sender/sender_mismatched_from_and_reply-to.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: "Sender: Mismatched From and Reply-to"
```
Comment (jkamdjou) [inline signals/sender/sender_mismatched_from_and_reply-to.yml:1]:
```suggestion
name: "Sender: Mismatched From and Reply-to Domain"
```

File: signals/sender/sender_unsolicited_sender_domain.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: "Sender: Unsolicited sender domain"
+type: "query"
+source: |
+  sender.email.email not in $recipient_emails
```
Comment (jkamdjou) [inline signals/sender/sender_unsolicited_sender_domain.yml:4]:
```suggestion
  sender.email.domain.domain not in $recipient_domains
```
guessing we also want one for email

File: signals/sender/sender_sender_is_custom_domain,_reply-to_is_freemail.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+name: "Sender: Sender is custom domain, reply-to is freemail"
+type: "query"
+source: |
+  any(headers.reply_to,
+          .email.domain.domain in $free_email_providers
+          and not .email.domain.domain == sender.email.domain.domain
```
Comment (jkamdjou) [inline signals/sender/sender_sender_is_custom_domain,_reply-to_is_freemail.yml:6]: freemail check instead

**Review by jkamdjou — APPROVED** [review]
LGTM - heads up this doesn't include Unsolicited sender email

**Pull request merged**